packages/components/src/assets/FadersHorizontal.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
```

Comment:
How about take the opportunity and uses the new approach (using `IconProps`) for icons assets? 

https://github.com/vtex/faststore/blob/main/packages/components/src/assets/House.tsx

Reply:
Nice! Let's leave this to another task so we can do it on all icons.